### PR TITLE
blacklist and whitelist changes to md and yml files

### PR DESCRIPTION
**Pull request by tschanfeld**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
[fixes: link to the issue](https://github.com/demisto/etc/issues/42124)

## Description
Goal is to replace blacklist/black list with block list and whitelist/white list with allow list, WITHOUT breaking anything.
Two concerns - a) that I could have changed text that will break something b) that we can end up with a confusing mix of blacklist and block list, whitelist and allow list, all in the same files, since some items did not look safe to change. I edited only md and yml files.

### CODE REVIEW

File: Packs/AbuseDB/Integrations/AbuseDB/AbuseDB.yml [hunk 1]
```diff
@@ -244,7 +244,7 @@ script:
     name: abuseipdb-get-blacklist
     outputs:
     - contextPath: AbuseIPDB.Blacklist
-      description: A list of blacklisted IP addresses.
+      description: A list of IP addresses on a block list.
```
Comment (yaakovi) [inline Packs/AbuseDB/Integrations/AbuseDB/AbuseDB.yml:247]: are we ok with mixed terminology in this case?
the path is called `blacklist` which we cannot change while the description is using `block list`.

**Comment by tschanfeld** [comment]
@ShahafBenYakir what is the status with this?

**Comment by ShahafBenYakir** [comment]
@tschanfeld In progress

**Comment by ShahafBenYakir** [comment]
Changes were only done in READMEs so Validation fails on unrelated issues

**Pull request merged**